scripts/globals/mobskills/super_buff.lua

```diff
@@ -0,0 +1,19 @@
+---------------------------------------------
+-- Spike Flail
```

Comment:
fix the comment for us if you would :+1:

Reply:
Will do.